src/payment/strategies/mollie/mollie-payment-strategy.ts

```diff
@@ -135,6 +157,55 @@ export default class MolliePaymentStrategy implements PaymentStrategy {
         return Promise.resolve(this._store.getState());
     }
 
+    private _isHostedFieldAvailable(options?: PaymentInitializeOptions): boolean {
+        if (!options) {
+            throw new InvalidArgumentError();
+        }
+
+        return (options.mollie?.containerId) ? true : false;
+    }
+
+    private  _mountCardVerificationfields(options: any): Promise<any> {
+        if (isEmpty(options.mollie.form)) {
+            return Promise.resolve();
+        }
+
+        const formOptions = options.mollie.form;
+        const { config } = this._store.getState();
+        const { paymentSettings: { bigpayBaseUrl: host = '' } = {} } = config.getStoreConfig() || {};
+
+        if (!formOptions) {
+            throw new InvalidArgumentError();
+        }
+
+        const form = this._hostedFormFactory.create(host, formOptions);
+
+        return new Promise(async (resolve, reject) => {
+            if (options.mollie.form) {
+                try {
+                    await form.attach();
+                    this._shouldRenderHostedForm = true;
+                    this._hostedForm = form;
+                } catch (error) {
+                    reject(new NotInitializedError(NotInitializedErrorType.PaymentNotInitialized));
+                }
+            }
+
+            resolve(form);
+        });
+    }
+
+    private _isHostedPaymentFormEnabled(methodId?: string, gatewayId?: string): boolean {
+        if (!methodId) {
+            return false;
```

Comment:
methodId should be mandatory. Or if it's missing, this should throw an error rather than return false.